src/utils/selectors/index.ts

```diff
@@ -68,6 +60,14 @@ export const pipeableSelect = (storeName: string) =>
 export const storeSelect = (storeName: string) =>
   `${storeExpression(storeName)}[callee.property.name='select']` as const
 
+export const storeDispatch = (storeName: string) =>
+  `${storeExpression(storeName)}[callee.property.name='dispatch']` as const
```

Comment:
I just noticed that we could reuse the `storeExpression` + I took the opportunity to rename this one to match the existing convention `store*`.